### ISSUE TITLE
Parallel view : Screen size limit -Book and chapter does not display alongside the selected language for languages with long string length

### ISSUE THREAD

**Issue by AnnCherianA** [Bridgeconn/VachanOnline-v2#44]
Issue description : On selecting a new language in the parallel view , the book and chapter does not display alongside the selected language for languages with long string length. (Malayalam, Deuteronomy, Nagamese, etc)
Steps:
Select a language and book (say Hindi IRV and यहोशू 20)
Now select parallel view and  select a language with long string such as Malayalam 
Observation:
Along with the selected language, the book  name and chapter does not display 
Expected : The code is to be enhanced to include a wider set of screen size resolution  such that the book and chapter name are displayed alongside the selected book 
![Missing book and chapter name in parallel bible view](https://user-images.githubusercontent.com/74647279/107216318-9d021300-6a32-11eb-98c9-e055a8c536a1.jpg)

**Comment by samuelanthrayose** [comment]
1. Reduce spacing between the book/chapter selector and note icon
2. Push all the icons to a menu on smaller screen sizes (separate feature)
3. Replace the book names with book abbreviations on smaller screen sizes (separate issue)

**Comment by AnnCherianA** [comment]
In parallel view, when long books are selected, the book dropdown is now visible.
![Parallel view screen length](https://user-images.githubusercontent.com/74647279/124512356-c2365a00-ddf5-11eb-88c5-13aea1b10149.jpg)
